### PR TITLE
Add support for Hindi language

### PR DESCRIPTION
**Pull request by rathodmitesh916**
#StayTheFuckHome
भाड़ में जाओ घर!

Used Google translate, ofc, for the translation word-to-word - must say it is pretty good by itself, mostly! 

Meanwhile reaching out to family & friends to give feedback.

Also addresses: https://github.com/flore2003/staythefuckhome/issues/43

### CODE REVIEW

**Comment by rathodmitesh916** [comment]
`Stay The Fuck Home!`

Loosely translates to

`भाड़ में जाओ घर!`

Better translations

`अपने कमबख्त घर में रहो!`
`घर पर रहने की कोशिश करो!` But looses the `f` point!

`कमबख्त घर में रहो!` - winner!

Screen Shot 2020-03-15 at 2 01 15 PM

**Comment by rathodmitesh916** [comment]
Hi @flore2003! Really like the domain and the content. Adding this PR to support for Hindi language. Let me know if needs to be updated. Thanks!

**Review by sthakuri — APPROVED** [review]
I am not a Hindi language native but I can read and speak Hindi well. Approving this request.

File: site/hi.njk [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+            <p>
+                अच्छी तरह से अध्ययन किए गए उपचार और एक व्यवहार्य वैक्सीन अभी भी कम से कम एक और वर्ष में होने के साथ, कोरोनोवायरस महामारी को खाड़ी में रखने का एकमात्र प्रभावी तरीका वायरस को फैलने की कम संभावना देना है।
+                निम्नलिखित कार्यों की सूची, जो महामारी के खिलाफ लड़ाई में सबसे प्रभावी को लागू करने के लिए सबसे आसान से आदेश दिया गया है, उन लोगों के लिए ढीले दिशानिर्देशों के एक सेट के रूप में काम करना चाहिए जो आंदोलन में शामिल होना चाहते हैं और कार्रवाई कर सकते हैं जो वास्तव में एक अंतर बना सकते हैं।
+            </p>
+
+            <ol class="text-xl mb-12 list-decimal pl-12">
+                <li class="pb-2">घबराओ मत, लेकिन सतर्क रहो।</li>
+                <li class="pb-2">अपने हाथों को अक्सर धोएं और अच्छी खांसी और छींकने का अभ्यास करें।</li>
+                <li class="pb-2">अपने मुँह, नाक और आँखों सहित जितना संभव हो सके अपने चेहरे को छूने की कोशिश करें।</li>
```
Comment (flore2003) [inline site/hi.njk]: I think this is missing the negative? Google translate translates it as: 'Try to touch your face as much as possible including your mouth, nose and eyes.' Sounds like bad advice :joy:

**Review by flore2003 — CHANGES_REQUESTED** [review]

**Comment by sthakuri** [comment]
yes,

अपने मुँह, नाक और आँखों सहित जितना संभव हो सके अपने चेहरे को छूने की
कोशिश करें <= this line has to be corrected.

File: site/hi.njk (hunk 1 above)
Comment (rathodmitesh916) [reply site/hi.njk]: THE MOST IMPORTANT REVIEW COMMENT OF MY PRS 2020

**Comment by rathodmitesh916** [comment]
Thanks for checking @flore2003 ! Updated the PR.

**Comment by rathodmitesh916** [comment]
@flore2003 GTM?

**Pull request closed without merging**